iothub/device/src/PlugAndPlayConvention/PnpConventionHelper.cs

```diff
@@ -50,6 +49,20 @@ public static Message CreateIothubMessageUtf8(string telemetryName, string seria
         /// <summary>
         /// Create a key-value property patch for both read-only and read-write properties.
```

Comment:
Propose:

> Create a key-value property patch for device properties.

And the in remarks note that this works for both read-only and read-write properties, both of which are named from a service perspective, and that to a device all properties are read-write.